### PR TITLE
GIX-1836: Add new primitive colors

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

We want to synchronize the colors in Figma and NNS Dapp.

In this PR, we introduce the color primitive used in Figma, but we don't use them yet.

Source of colors: https://www.figma.com/file/P4JbeFs6oGt2WmKeB4PHFc/Updated-NNS-Styles-%26-Components?type=design&node-id=514-220550&mode=design&t=4slnxbCcIfhrcgu6-0

Under Local Variables > Primitives

# Changes

* New styles global file color.scss
* Add new file to global imports.

# Screenshots

Nothing changed yet.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @mstrasinskis please review

File: src/lib/styles/global/color-primitives.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  --purple-dark-400: #483666;
+  --purple-dark-450: #3d2a5c;
+  --purple-dark-500: #372652;
+  --purple-dark-550: #31204c;
+  --purple-dark-600: #2d1a4a;
+  --purple-dark-650: #291942;
+  --purple-dark-900: #24143d;
+
+  // Accessory
+  --fade: rgba(36, 20, 61, 0.6); // #24143d 60%
```
Comment (mstrasinskis) [inline src/lib/styles/global/color-primitives.scss:32]: What is this colour for? I didn't find it in Figma.

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks!

File: src/lib/styles/global/color-primitives.scss (hunk 1 above)
Comment (mstrasinskis) [reply src/lib/styles/global/color-primitives.scss:32]: If there will be some derived colours, maybe `--fade` should live there?

Comment (mstrasinskis) [reply src/lib/styles/global/color-primitives.scss:32]: `rgba(var(--purple-dark-900), 0.6)`?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks.

A suggestion if you would like it: `color-primitives.scss` can also be shortened to `colors.scss`

**Comment by lmuntaner** [comment]
> LGTM, thanks.
> 
> A suggestion if you would like it: `color-primitives.scss` can also be shortened to `colors.scss`

I wasn't sure with the name, but I also prefer it shorter.

File: src/lib/styles/global/color-primitives.scss (hunk 1 above)
Comment (lmuntaner) [reply src/lib/styles/global/color-primitives.scss:32]: I don't know, but it's a good question. I'll ask it to Artem and then change based on the feedback.

The derived colors will be in each theme. I will ask him where we should put those. I agree that maybe this one belongs there if it's a derived color.

**Pull request merged**